backend/src/routes/nativeLanguages.js

```diff
@@ -0,0 +1,49 @@
+
+const express = require('express')
+const router = express.Router()
+const debug = require('debug')('app:nativeLanguages')
+const db = require('../models/index')
+const nativeLanguages = require('../helpers/nativeLanguages')
+
+const { UniqueViolationError } = require('objection')
+
+/* GET nativeLanguages listing. */
+router.get('/', async (req, res) => {
+  debug('Hello World!')
```

Comment:
I think we can clean up these debug calls.

Reply:
Okay, done!